aws/acm_test.go

```diff
@@ -69,9 +71,64 @@ func TestACM(t *testing.T) {
 				Error:       nil,
 			},
 		},
+		{
+			msg: "Found ACM Cert with correct filter tag",
```

Comment:
Would you like to add tests with several certificates in the list.
For example, "2 certs with corresponding tag" or "1 cert with corresponding tag + 1 cert with not corresponding tag"?